src/components/DropDown/DropDown.tsx

```diff
@@ -0,0 +1,35 @@
+import { ChangeEvent, useMemo, memo } from 'react';
+import cls from './DropDown.module.scss';
+
+interface IDropDownProps {
+  values?: string[];
+  onChange?: (value: string) => void;
+}
+
+export const DropDown = memo((props: IDropDownProps) => {
```

Comment:
I'm not sure the memo hook works correctly, espesially you throw function in props. Please check do you use useCallback for `onChange` function